examples/nirfmxlte/ul-modacc-single-carrier.py

```diff
@@ -0,0 +1,289 @@
+    )
+    raise_if_error(
+        client.CfgDuplexScheme(
+            nirfmxlte_types.CfgDuplexSchemeRequest(
+                instrument=instr,
+                selector_string="",
+                duplex_scheme=nirfmxlte_types.DUPLEX_SCHEME_FDD,
+                uplink_downlink_configuration=nirfmxlte_types.UPLINK_DOWNLINK_CONFIGURATION_0,
+            )
+        )
+    )
+    raise_if_error(
+        client.CfgAutoDMRSDetectionEnabled(
+            nirfmxlte_types.CfgAutoDMRSDetectionEnabledRequest(
+                instrument=instr,
+                selector_string="",
+                auto_dmrs_detection_enabled=nirfmxlte_types.AUTO_DMRS_DETECTION_ENABLED_TRUE,
+            )
+        )
+    )
+    raise_if_error(
+        client.SelectMeasurements(
+            nirfmxlte_types.SelectMeasurementsRequest(
+                instrument=instr,
+                selector_string="",
+                measurements=nirfmxlte_types.MEASUREMENT_TYPES_MODACC,
+                enable_all_traces=True,
+            )
+        )
+    )
+    raise_if_error(
+        client.ModAccCfgSynchronizationModeAndInterval(
+            nirfmxlte_types.ModAccCfgSynchronizationModeAndIntervalRequest(
+                instrument=instr,
+                selector_string="",
+                synchronization_mode=nirfmxlte_types.MODACC_SYNCHRONIZATION_MODE_SLOT,
+                measurement_offset=0,
+                measurement_length=1,
+            )
+        )
+    )
+    raise_if_error(
+        client.ModAccCfgEVMUnit(
+            nirfmxlte_types.ModAccCfgEVMUnitRequest(
+                instrument=instr,
+                selector_string="",
+                evm_unit=nirfmxlte_types.MODACC_EVM_UNIT_PERCENTAGE,
+            )
+        )
+    )
+    raise_if_error(
+        client.ModAccCfgInBandEmissionMaskType(
+            nirfmxlte_types.ModAccCfgInBandEmissionMaskTypeRequest(
+                instrument=instr,
+                selector_string="",
+                in_band_emission_mask_type=nirfmxlte_types.MODACC_IN_BAND_EMISSION_MASK_TYPE_RELEASE_11_ONWARDS,
+            )
+        )
+    )
+    raise_if_error(
+        client.ModAccCfgAveraging(
+            nirfmxlte_types.ModAccCfgAveragingRequest(
+                instrument=instr,
+                selector_string="",
+                averaging_enabled=nirfmxlte_types.MODACC_AVERAGING_ENABLED_FALSE,
+                averaging_count=10,
+            )
+        )
+    )
+    raise_if_error(
+        client.Initiate(
+            nirfmxlte_types.InitiateRequest(instrument=instr, selector_string="", result_name="")
+        )
+    )
+
+    mod_acc_fetch_composite_evm_response = raise_if_error(
+        client.ModAccFetchCompositeEVM(
+            nirfmxlte_types.ModAccFetchCompositeEVMRequest(
+                instrument=instr, selector_string="", timeout=10.000000
+            )
+        )
+    )
+    mean_rms_composite_evm = mod_acc_fetch_composite_evm_response.mean_rms_composite_evm
```

Comment:
I feel like it would be cleaner to simply inline this when we print, it would at least reduce the line length and distance between these variables are actually used

Reply:
This is similar to the other one: we don't actually print several of these extracted values; I'm just showing that they can be extracted from the response. In that respect, it makes more sense to me for them to be extracted next to where the response is received. But if we don't care about extracting those extra unprinted values, then I'll make this change, too.

Reply:
All of the ones from line 222-251 are being printed. I don't have a strong opinion either way though